modules/large-deployments/pages/hub-namespaces.adoc

```diff
@@ -1,36 +1,36 @@
 [[lsd-hub-namespaces]]
 = Hub XMLRPC API Namespaces
 
-Hub XMLRPC API operates in a similar way to the {productname} API.
+The Hub XMLRPC API operates in a similar way to the {productname} API.
 For {productname} API documentation, see https://documentation.suse.com/suma.
 
 The Hub XMLRPC API exposes the same methods that are available from the server's XMLRPC APIs, with a few differences in parameter and return types.
 Additionally, the Hub XMLRPC API supports some Hub-specific end points which are not available in the {productname} API.
 
-Hub supports three different namespaces:
+The Hub XMLRPC API supports three different namespaces:
 
-* The ``hub`` namespace is used to target the Hub XMLRPC API Server.
-It supports all the methods that are available on a {productname} Server XMLRPC API as well as Hub-specific XMLRPC endpoints.
+* The ``hub`` namespace is used to target the Hub-specific XMLRPC endpoints.
 Hub-specific methods are primarily related to authentication.
 * The ``unicast`` namespace is used to target a single peripheral server registered in the hub.
 It redirects any call transparently to one specific server and returns any value as if the server's XMLRPC API endpoint was used directly.
 * The ``multicast`` namespace is used to target multiple peripheral servers registered in the hub.
 It redirects any call transparently to all the specified servers and returns the results in the form of a ``map``.
+* It is possible to target directly the underlying XMLRPC API of the Hub server, given that the HUB XMLRPC API exposes all the methods that are available on a {productname} Server XMLRPC API.
```

Comment:
What is the point that you are trying to make here? The list is supposed to contain the three namespaces ("The Hub XMLRPC API supports three different namespaces").

Reply:
Yeah, I can't find the right words for this.

My point is that the user can authenticate to the hub XMLRPC API, using, eg, client.hub.Login("user","pass")

and then you can:
- use the hub-specific endpoints (eg, client.hub.listServerIds)
- use the multicast
- use the unicast
- AND call the Uyuni XMLRPC of the hub server directly, with the same client, and it does not require any special namespace. For example, you can have a statement like 'client.system.listSystems(hubSessionKey)', which will execute the  'system.listSystems' method of the Uyuni XMLRPC API of the Hub server

Reply:
TBH, I'm not sure if it's worth to mention this in the doc

Reply:
Let me try to reformulate...

```suggestion
* methods called without any of the above namespaces will be forwarded to the normal XMLRPC API of the Hub (ie. the one exposed on ports 80 and 443)
```

Reply:
thanks!